datadome.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "rack"
   spec.add_runtime_dependency "faraday", ">= 0.9.2"
 
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "bundler", "~> 2.2.24"
```

Comment:
`~> 2.2` please, we don't care about the minor version.

Reply:
ok.